src/pandas_profiling/config_default.yaml

```diff
@@ -90,23 +90,23 @@ missing_diagrams:
 
 correlations:
```

Comment:
@jtook is this configuration also reflected for the Tsmode?

Reply:
Yes, so this is reflected when tsmode is enabled and the default configuration file is specified